vagrant/provisioning/roles/privacy/tasks/main.yml

```diff
@@ -93,20 +93,20 @@
     regexp: "acm-activemq"
     replace: "{{ activemq_host }}"
 
-#- name: configure LDAP
-#  include_role: 
-#    name: arkcase-ldap-config
-#  vars:
-#    old_config_id: privacyportal
-#    new_config_id: "privacyportal"
-#    autogenerate_user_id: true
-#    ldap_user_domain_bean_name: "privacyportal"
-#    ldap_authentication_service_class: "gov.privacy.service.SARLdapAuthenticationService"
-#    portal_prefix: "portal."
-#    ldap_template_name: "spring-config-privacyportal-ldap.xml"
-#    group_base: "{{ ldap_portal_group_no_base | default(ldap_group_no_base) }}"
-#    user_base: "{{ ldap_portal_user_no_base | default(ldap_user_no_base) }}"
-#  when: encrypted_ldap_bind_password is defined
+- name: configure LDAP
+  include_role: 
+    name: arkcase-ldap-config
+  vars:
+    old_config_id: privacyportal
+    new_config_id: "privacyportal"
+    autogenerate_user_id: true
+    ldap_user_domain_bean_name: "privacyportal"
+    ldap_authentication_service_class: "gov.privacy.service.SARLdapAuthenticationService"
+    portal_prefix: "portal."
+    ldap_template_name: "spring-config-privacyportal-ldap.xml"
+    group_base: "{{ ldap_portal_group_no_base | default(ldap_group_no_base) }}"
```

Comment:
I suppose the privacy portal and the FOIA portal should have different OUs in case they are both being used in the same installation.

Reply:
We don't have that option right now. Privacy is a CaseFile extension, as is Foia. I don't think we can enable them both.

Reply:
OK .. we'll worry about this when we actually have this requirement.